src/ansys/fluent/core/solver/allowed_name_error_message.py

```diff
@@ -0,0 +1,4 @@
+from ..allowed_name_error_msg import (  # noqa: F401
```

Comment:
Should we have just one file error_msg.py that contains classes for different errors, instead of allowed_name_error_msg.py, allowed_name_error_message.py etc.?

But why this file, in any case?

Reply:
Yes, definitely. We will keep single `error_msg.py` file.